homeassistant/components/tessie/binary_sensor.py

```diff
@@ -0,0 +1,146 @@
+"""Binary Sensor platform for Tessie integration."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorDeviceClass,
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import TessieDataUpdateCoordinator
+from .entity import TessieEntity
+
+PARALLEL_UPDATES = 0
+
+
+@dataclass
+class TessieBinarySensorEntityDescription(BinarySensorEntityDescription):
+    """Describes Tessie binary sensor entity."""
+
+    is_on: Callable[..., bool] = lambda x: x
+
+
+DESCRIPTIONS: tuple[TessieBinarySensorEntityDescription, ...] = (
```

Comment:
I think for the strings we want all snake_case, for keys it doesn't matter, but for translation keys it does iirc

Reply:
So you want me to use _ instead of - right?

Reply:
This has been changed